src/go/k8s/kuttl-test.yaml

```diff
@@ -8,6 +8,8 @@ testDirs:
   - ./tests/e2e
 kindConfig: ./kind.yaml
 commands:
-  - command: "kubectl apply -k config/default"
+  - command: "./hack/install-cert-manager.sh"
```

Comment:
For a non-testing deployments, would the cert-manager and the verifier be deployed through the operator helm chart?

Reply:
verifier does not need to be deployed, that's binary used locally to verify the environment. Yes, cert-manager is dependency of the operator right now.

Reply:
I think `cert-manager` should be included in helm-chart as a optional dependency. Users that creates k8s cluster will need cert-manager for many things so they can use one `cert-manager` for all workloads. Our operator deployment uses custom resources and it requires `cert-manager` to fulfil our desire state.